frontend/components/base/data-table/data-table.tsx

```diff
@@ -379,29 +385,44 @@ function DataTable<T>({
   }, [sorting]);
 
   return (
-    <>
+    <div className={cn('w-full', isFullscreen && 'absolute top-0 right-0 bottom-0 left-0 bg-white z-50 p-4')}>
```

Comment:
@lflangis Les deux sont valide mais de cette manière ca evite de mettre un false dans le className.

```suggestion
    <div className={cn('w-full', {
"absolute top-0 right-0 bottom-0 left-0 bg-white z-50 p-4": isFullScreen
})}>
```

Reply:
En effet, j'aime mieux cette syntaxe. Je vais mettre a jour